requirements.txt

```diff
@@ -1,2 +1,4 @@
 numpy
 xarray
+dask[array]
```

Comment:
And could you please add all new requirements to `additional_dependencies` in `.pre-commit-config.yaml`, see #39 for context?

Reply:
Yep, what is the scope of those dependencies though?  Is there any reason it wouldn't always be an exact copy of what's in `requirements.txt`?

Reply:
precommit creates a separate venv for the hooks, so for mypy to work we need to install the requirements there as well, and right now there is not way to tell precommit to use requirements file.